src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,10 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
```

Comment:
![image](https://github.com/mate-academy/jv-oop/assets/49767049/42ab0708-5b1c-4a64-9a2b-63be40072584)
\+ you can init this array like this
![image](https://github.com/mate-academy/jv-oop/assets/49767049/6775509a-2fe2-458b-a96a-6c8d7cdcc9fd)